deployment/clouddeploy/gke-workers/base/exporter.yaml

```diff
@@ -3,9 +3,9 @@ kind: CronJob
 metadata:
   name: exporter
   labels:
-    cronLastSuccessfulTimeMins: "90"
+    cronLastSuccessfulTimeMins: "180"
 spec:
-  schedule: "*/30 * * * *"
```

Comment:
I think it's fine to keep it every `*/30`, as we've set the concurrency policy to 'forbid'. Setting `*/30` gives us more frequent runs than `*/60` if the cron job takes just slightly over 60 minutes.

Reply:
Okay, d17ae0f7 leaves the run frequency at 30 minutes.